arrays/victor/C#_soft-188/5_task/5_task/5_task/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Linq;
+
+namespace _5_task
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] mas = { 1, 2, 5, 78, 2, 34, 248, 11, 9, 29, 0, 270 };
+            Console.WriteLine("Масив = 1, 2, 5, 78, 2, 34, 248, 11, 9, 29, 0, 270");
+            int sum = 0;
+            int kl = 1;
+
+            for (int i = 0; i < mas.Length; i++)
+            {
+                if (mas[i] %2 == 0)
+                {
+                    sum += mas[i];
+                }
+            }
+
+            for (int j = 0; j < mas.Length; j++)
+            {
```

Comment:
Quantity of elements is wrong. 
kl = 0 not 1
and you can simplify this 
if (mas[i] %2 == 0)
{
       sum += mas[i];
       kl++;
}